randovania/data/json_data/prime1.txt

```diff
@@ -4510,7 +4554,9 @@ Asset id: 1014518864
 
```

Comment:
Possibly rename the Missiles in Ruined Shrine to be more descriptive based on their locations?